tools/cldr-apps/js/src/cldrVueRouter.js

```diff
@@ -92,10 +95,28 @@ function createCldrApp(component, specialPage, extraProps) {
  * @returns {App} the App object
  */
 function show(component, el, specialPage, extraProps) {
-  const app = createCldrApp(component, specialPage, extraProps);
 
-  // Fire up the app..
-  lastRoot = app.mount(el);
+  function errBox(e, operation) {
+    console.error(`There was a problem ${operation} “${specialPage}”`);
+    console.error(e);
+    notification.error({
+      message: `Problem ${operation} “${specialPage}”`,
+      description: `${e.message} — (click to reload)`,
+      placement: "topLeft",
+      duration: 0,  // do not auto close
+      onClick: () => window.location.reload(),
+    });
+  }
+
+  let app;
+  try {
+    app = createCldrApp(component, specialPage, extraProps);
+    lastRoot = app.mount(el);
+  } catch (e) {
+    // did we successfully create the app?
+    errBox(e, (!!app)?"showing":"creating");
+    return null;
```

Comment:
needs formatting

I would move errBox to below its caller, and not make it nested

in the long run such a function might be more at home in cldrRetry.js for re-use

otherwise LGTM